src/main/cli/configs.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+from enum import Enum
+from typing import List
+
+from src.main.preprocessing.inefficient_statements_removing import remove_inefficient_statements
+from src.main.preprocessing.int_experience_adding import add_int_experience
+from src.main.preprocessing.intermediate_diffs_removing import remove_intermediate_diffs
+from src.main.preprocessing.preprocessing import preprocess_data
+from src.main.splitting.splitting import split_tasks_into_separate_files
+from src.main.splitting.tasks_tests_handler import run_tests
+
+
+class PLOT_TYPE(Enum):
+    PARTICIPANTS_DISTRIBUTION = 'participants_distr'
+    TASKS_DISTRIBUTION = 'tasks_distr'
+    SPLITTING_PLOTS = 'splitting_plots'
+
+    @classmethod
+    def plot_types(cls) -> List['PLOT_TYPE']:
+        return [_ for _ in PLOT_TYPE]
+
+    @classmethod
+    def values(cls) -> List[str]:
+        return [p_t.value for p_t in PLOT_TYPE]
+
+    @classmethod
+    def description(cls) -> str:
+        return f'{PLOT_TYPE.PARTICIPANTS_DISTRIBUTION.value} - visualize participants distribution; ' \
+               f'{PLOT_TYPE.TASKS_DISTRIBUTION.value} - visualize tasks distribution; ' \
+               f'{PLOT_TYPE.SPLITTING_PLOTS.value} - visualize splitting plots;'
+
+    @classmethod
+    def str_to_plot_type(cls, value: str) -> 'PLOT_TYPE':
```

Comment:
It's better to use `from __future__ import annotations` instead of writing quotes: it works faster, it's a more modern way to solve a problem with forwarding references since Python 3.7 and it will be used in a future Python releases.